code/modules/projectiles/guns/projectile/revolver/wayfarer.dm

```diff
@@ -0,0 +1,24 @@
+//Sprite by Rebel
+/obj/item/gun/projectile/revolver/wayfarer
+	name = "\"Wayfarer\" caseless revolver"
+	desc = "A exstreamly well hand made revolver of a mix of steel, iron and plasteel with a polished wooden stock. \
+	The barrel inside it refined to be ideal for 10x24 caseless and easy to clean. \
```

Comment:
"The Barrel is refined to use high-power 10x24 caseless munitions while being easy to clean."